### ISSUE TITLE
Upgrade anymatch

### ISSUE THREAD

**Issue by mihkeleidast** [frctl/fractal#674]
### What problem would this feature solve?

Currently, when installing Fractal, users get a few npm deprecation warnings:
```
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
```
These warnings come via our use of anymatch:
```
└─┬ @frctl/fractal@1.4.1
  └─┬ anymatch@2.0.0
    └─┬ micromatch@3.1.10
      └─┬ snapdragon@0.8.2
        └─┬ source-map-resolve@0.5.3
          └── resolve-url@0.2.1 
          └── urix@0.1.0
```

### What the feature should look like?

Fractal uses updated anymatch that does not depend on deprecated packages anymore.

Note that I filed this as a separate issue since some tests fail & some test files are discovered as components etc - so a bit more work here than just updating.

**Comment by mihkeleidast** [comment]
Investigated this a bit, it sure looks like the mismatching of files is a bug in anymatch - I filed micromatch/anymatch#36 to get some feedback from them.

**Comment by mihkeleidast** [comment]
Resolved in #970.
